torch_em/data/datasets/monuseg.py

```diff
@@ -45,14 +95,35 @@ def get_monuseg_dataset(
 
 # TODO implement selecting organ
 def get_monuseg_loader(
-    path, patch_shape, batch_size, download=False, offsets=None, boundaries=False, binary=False, **kwargs
+    path, patch_shape, batch_size, split, download=False, offsets=None, boundaries=False, binary=False, **kwargs
 ):
     ds_kwargs, loader_kwargs = util.split_kwargs(
         torch_em.default_segmentation_dataset, **kwargs
     )
     dataset = get_monuseg_dataset(
-        path, patch_shape, download=download,
+        path, patch_shape, split, download=download,
         offsets=offsets, boundaries=boundaries, binary=binary, **ds_kwargs
     )
     loader = torch_em.get_data_loader(dataset, batch_size, **loader_kwargs)
     return loader
+
+
+def main():
```

Comment:
This is not a good practice. Libraries should not contain scripts inside. Instead add a small script to test it here: https://github.com/constantinpape/torch-em/tree/main/scripts/datasets

Reply:
Thanks for the heads up. Would check all the dataloaders in scripts from now.